lib/resty/mlcache/ipc.lua

```diff
@@ -13,6 +13,7 @@ local find         = string.find
 local min          = math.min
 local type         = type
 local pcall        = pcall
+local error        = error
```

Comment:
Yes, I saw it on `mlcache` and decided that it looks better if we are consistent.